enterprise/migrations/0175_auto_20230627_2027.py

```diff
@@ -0,0 +1,306 @@
+# Generated by Django 3.2.19 on 2023-06-27 20:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+import enterprise.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sites', '0002_alter_domain_unique'),
+        ('enterprise', '0174_auto_20230608_2041'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='active',
+            field=models.BooleanField(default=True, verbose_name='Active admin portal'),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='auth_org_id',
+            field=models.CharField(blank=True, help_text="Enterprise customer's authentication ID if leveraging a third party platform such as Auth0 for authentication.", max_length=80, null=True, verbose_name='Third Party Auth Org ID:'),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='contact_email',
+            field=models.EmailField(blank=True, help_text='Email address presented on learner portal as public point of contact from customer organization.', max_length=254, null=True, verbose_name='Customer admin contact email:'),
```

Comment:
Ask Mike about the colons? I think it's weird that he has them on some fields and not others in the Figma and want to make they are intentional

Reply:
Settings that had a dropdown or free text field have a colon, where checkboxes did not. However, Mike told me he missed a couple of the colons in the wireframe for the options with dropdowns/text fields. He just added two colons back to a couple of those options in the wireframe.